index.html

```diff
@@ -1,13 +1,29 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <title></title><link rel="stylesheet" href="style.css">
+    <title>Salmon Cookies</title><link rel="stylesheet" href="mainstyle.css">
```

Comment:
These should be on different lines, and I would put the charset meta tag back in.